org.wezfurlong.wezterm.json

```diff
@@ -0,0 +1,66 @@
+{
+    "app-id": "org.wezfurlong.wezterm",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "21.08",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command": "wezterm",
+    "finish-args": [
+        "--share=ipc",
+        "--filesystem=home:ro",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--talk-name=org.freedesktop.Flatpak",
+        "--talk-name=org.freedesktop.Notifications",
+        "--talk-name=org.freedesktop.portal.Desktop",
+        "--talk-name=org.freedesktop.portal.Settings",
+        "--share=network"
+    ],
+    "build-options": {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin"
```

Comment:
this could be moved to the build options below inside the module. This is just cosmetic.